<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapGenerateOptions</class>
 <widget class="QWidget" name="MapGenerateOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>630</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>862</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Map Generate Options</string>
  </property>
  <widget class="QLabel" name="modeLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择游戏模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="modeButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>模式: 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelX">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>宽度 (默认 20)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelY">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>长度 (默认 20)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeeds">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>种子:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>随机种子 (默认当前时间)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Nest">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>迷宫是否允许嵌套</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
